windowControlsOverlay/index.html

```diff
@@ -22,6 +22,8 @@
         <p id="WCO" class="mono">navigator.windowControlsOverlay</p>
         <p id="WCOVisible" class="mono"></p>
         <div id="WCORect" class="mono"></div>
+        <div id="WCOCss" class="mono"></div>
+        <div id="WCOCssTest" class="mono"></div>
```

Comment:
Can you pull this out of this section or at least remove class=mono? It looks like it should have text (since that's what is here) and I worry that I'll accidently delete it later and break the CSS values

Reply:
removed `class=mono` :)